operations/helm/charts/grafana-agent/templates/rbac.yaml

```diff
@@ -23,6 +23,33 @@ rules:
       - get
       - list
       - watch
+  # Rules which allow loki.source.kubernetes and loki.source.podlogs to work.
+  - apiGroups:
+      - ""
+    resources:
+      - pods
+      - pods/log
```

Comment:
I triple checked this, but it is in fact `pods/log` and not `pod/logs` or `pods/logs`.